walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -19,5 +19,15 @@
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
     throw new UnsupportedOperationException("TODO: Implement this method.");
+    //create an empty Collection<TimeRange> for available times for the requested meeting
+    //check if the duration of the event is more than a day or if the collection of attendees is empty
+    //if yes, return the empty Collection
+    //if no, begin looking for appropriate time slots
+    //Iterate through the day by checking each appropriate time duration ex. if meeting duration is 1hr 8-9, 8:30-9:30, ...
+    //for each time duration check if there is an event present and if anyone is already attending that meeting
```

Comment:
I think you should add detail about how you will check whether there is an existing event present that overlaps with the possible new meeting time, since the input is a pretty plain `Collection<Event>` rather than a fancy class with a `.getOverlappingMeetings(TimeRange)` or something built in.